bin/rbenv-ctags

```diff
@@ -17,6 +17,16 @@ generate_ctags_in() {
   (cd "$1"; ctags -R --languages=ruby)
 }
 
+generate_c_ctags_in() {
+  if [ -n "$RUBY_BUILD_BUILD_PATH" ]; then
+    echo "Running ctags on $RUBY_BUILD_BUILD_PATH"
+    (cd "$1"; ctags --languages=C,C++ -R "$RUBY_BUILD_BUILD_PATH")
+  else
+    echo "Running ctags on $1"
+    (cd "$1"; ctags -R --languages=C,C++)
```

Comment:
You can consolidate both conditional branches with this single line:

```sh
ctags -R --languages=C,C++ ${RUBY_BUILD_BUILD_PATH:+"$RUBY_BUILD_BUILD_PATH"}
```

i.e. conditionally passing an extra parameter based on whether it's empty or not.